spring-cloud-dataflow-server-local/pom.xml

```diff
@@ -17,6 +17,24 @@
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-starter-dataflow-server-local</artifactId>
+			<exclusions>
+				<exclusion>
+					<groupId>commons-io</groupId>
+					<artifactId>commons-io</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+		<dependency>
+			<groupId>commons-io</groupId>
+			<artifactId>commons-io</artifactId>
+			<version>2.4</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.directory.server</groupId>
+			<artifactId>apacheds-protocol-ldap</artifactId>
+			<version>1.5.5</version>
+			<scope>test</scope>
```

Comment:
That is no purpose. I am doing full-blown end-to-end testing bootstrapping an entire SCDF server instance. This also allows me to take into account the configuration file from the `spring-cloud-starter-dataflow-server-local` project. 

Ultimately, I am glad that I did that, as that highlighted and issue when enabling security in conjunction with the Boot management REST endpoints.

Lastly, the infrastructure here, might also be used (and refactored) for using in the other implementations such as CF, Yarn etc. Also, being able to bootstrap an entire SCDF instance,  might also be a useful spot to run the Spring Rest Doc tests.